util/eventbus/bus.go

```diff
@@ -65,12 +66,17 @@ func (b *Bus) Client(name string) *Client {
 		bus:  b,
 		pub:  set.Set[publisher]{},
 	}
-	b.topicsMu.Lock()
-	defer b.topicsMu.Unlock()
+	b.clientsMu.Lock()
+	defer b.clientsMu.Unlock()
 	b.clients.Add(ret)
 	return ret
 }
 
+// Debugger returns the debugging facility for the bus.
+func (b *Bus) Debugger() Debugger {
+	return Debugger{b}
```

Comment:
Not an action item for this PR, but it occurs to me we could keep a gauge for how many of these are "open", and surface that in the debug view as a way of keeping tabs on expected use. Later, when we add metrics.